docs/DATA_FORMATS_INPUT.md

```diff
@@ -115,6 +115,10 @@ For example, if you had this configuration:
 
   ## measurement name suffix (for separating different commands)
   name_suffix = "_mycollector"
+  
+  ## specifies if the incoming JSON data is an array of metric data (true) 
+  ## to parse or a single object (false)
+  array = false
```

Comment:
I would actually probably recommend just scanning the bytes until you find the first occurence of either [ or {, and then call the appropriate parsing function based on that.